oxidation/crates/parsec_api_protocol/src/cmds.rs

```diff
@@ -0,0 +1,205 @@
+        (crate::organization, organization_config, OrganizationConfig), // organization_config has been added in api v2.2
+        // Realm
+        (crate::realm, realm_create, RealmCreate),
+        (crate::realm, realm_stats, RealmStats),
+        (crate::realm, realm_status, RealmStatus),
+        (
+            crate::realm,
+            realm_get_role_certificates,
+            RealmGetRoleCertificates
+        ),
+        (crate::realm, realm_update_roles, RealmUpdateRoles),
+        (
+            crate::realm,
+            realm_start_reencryption_maintenance,
+            RealmStartReencryptionMaintenance
+        ),
+        (
+            crate::realm,
+            realm_finish_reencryption_maintenance,
+            RealmFinishReencryptionMaintenance
+        ),
+        // User
+        (crate::user, user_get, UserGet),
+        (crate::user, user_create, UserCreate),
+        (crate::user, user_revoke, UserRevoke),
+        (crate::user, device_create, DeviceCreate),
+        // Human
+        (crate::user, human_find, HumanFind),
+        // Vlob
+        (crate::vlob, vlob_poll_changes, VlobPollChanges),
+        (crate::vlob, vlob_create, VlobCreate),
+        (crate::vlob, vlob_read, VlobRead),
+        (crate::vlob, vlob_update, VlobUpdate),
+        (crate::vlob, vlob_list_versions, VlobListVersions),
+        (
+            crate::vlob,
+            vlob_maintenance_get_reencryption_batch,
+            VlobMaintenanceGetReencryptionBatch
+        ),
+        (
+            crate::vlob,
+            vlob_maintenance_save_reencryption_batch,
+            VlobMaintenanceSaveReencryptionBatch
+        ),
+    ]
+);
+
+cmds_bundle!(
+    invited_cmds,
+    [
+        (crate::ping, ping, InvitedPing, "ping"),
+        (crate::invite, invite_info, InviteInfo),
+        (
+            crate::invite,
+            invite_1_claimer_wait_peer,
+            Invite1ClaimerWaitPeer,
+            "invite_1_claimer_wait_peer",
+        ),
+        (
+            crate::invite,
+            invite_2a_claimer_send_hashed_nonce_hash_nonce,
+            Invite2aClaimerSendHashedNonceHashNonce,
+            "invite_2a_claimer_send_hashed_nonce_hash_nonce",
+        ),
+        (
+            crate::invite,
+            invite_2b_claimer_send_nonce,
+            Invite2bClaimerSendNonce,
+            "invite_2b_claimer_send_nonce",
+        ),
+        (
+            crate::invite,
+            invite_3a_claimer_signify_trust,
+            Invite3aClaimerSignifyTrust,
+            "invite_3a_claimer_signify_trust",
+        ),
+        (
+            crate::invite,
+            invite_3b_claimer_wait_peer_trust,
+            Invite3bClaimerWaitPeerTrust,
+            "invite_3b_claimer_wait_peer_trust",
+        ),
+        (
+            crate::invite,
+            invite_4_claimer_communicate,
+            Invite4ClaimerCommunicate,
+            "invite_4_claimer_communicate",
+        ),
+    ]
+);
+
+#[test]
```

Comment:
What do you think of moving this test within the macro definition so that we run this check for each command !

I think it's interesting to have this test even if the `impl_dumps_loads_for_rep` already fail to compile if `UnknownError` is not present given it makes it very explicit that `UnknownError` must be defined for each rep.

Reply:
Done :+1: